src/Exceptions/HttpHTTPVersionNotSupported.php

```diff
@@ -2,10 +2,10 @@
 
 namespace Tnapf\Router\Exceptions;
 
-class HttpHTTPVersionNotSupported extends HttpException {
+class HttpHTTPVersionNotSupported extends HttpException
```

Comment:
Naming scheme doesn't really work for this one :^)

I'd just go for a namespace like `Tnapf\Router\Exceptions\Http` rather than prefixing the name, usually a better solution anyway

Reply:
Didn't notice this weird occurrence either 🫠